common/src/main/java/org/opensearch/ml/common/transport/connector/MLCreateConnectorInput.java

```diff
@@ -133,12 +149,15 @@ public static MLCreateConnectorInput parse(XContentParser parser) throws IOExcep
                 case ACCESS_MODE_FIELD:
                     access = AccessMode.from(parser.text());
                     break;
+                case DRY_RUN_FIELD:
+                    dryRun = parser.booleanValue();
```

Comment:
dryRun is mandatory in request?

Reply:
not mandatory, its default value is false, so it's ok to not use optional method.